app/scripts/services/dimInfoService.factory.js

```diff
@@ -8,14 +8,14 @@
 
   function InfoService(toaster, $http, SyncService) {
     return {
-      show: function(id, content) {
+      show: function(id, content, timeout = 0) {
```

Comment:
We're not quite in ES6-land yet, so we're not using things like default values. Would you mind doing this with just `timeout || 0`?